Tasks/MavenAuthenticateV0/taskJsonOverride.json

```diff
@@ -0,0 +1,51 @@
+{
+    "inputs": [
+      {
+        "name": "workloadIdentityServiceConnection",
+        "label": "'Entra Workload ID-backed Azure DevOps user' Service Connection",
+        "helpMarkDown": "This is an Entra Workload ID-backed Azure DevOps user Service Connection. If this is set, feedUrl and feedName are required. All other inputs are ignored.",
+        "type": "connectedService:workloadidentityuser",
+        "required": false,
+        "properties": {
+          "EditableOptions": "False",
+          "MultiSelectFlatList": "False"
+        }
+      },
+      {
+        "name": "feedUrl",
+        "label": "Azure Artifacts Feeds url.",
+        "helpMarkDown": "This is the packaging url of the feed your service principal should connect to. If this is set, workloadIdentityServiceConnection and feedName are required. All other inputs are ignored. Not compatible with mavenServiceConnections.",
```

Comment:
Provide an example, check npm's markdown

Reply:
Changed this!